include/sys/zio_compress.h

```diff
@@ -134,13 +136,16 @@ typedef int zio_decompresslevel_func_t(void *src, void *dst,
 typedef int zio_getlevel_func_t(void *src, size_t s_len, uint8_t *level);
 
 
+#if !defined(_STANDALONE)
 /*
  * Common signature for all zio decompress functions using an ABD as input.
  * This is helpful if you have both compressed ARC and scatter ABDs enabled,
  * but is not a requirement for all compression algorithms.
  */
 typedef int zio_decompress_abd_func_t(abd_t *src, void *dst,
     size_t s_len, size_t d_len, int);
+#endif
```

Comment:
Does leaving this functional available cause a problem?  Perhaps some kind of conflict with the abd type?

Reply:
I think there was a conflict. Same answer.

Reply:
There was a type conflict with abd_t. I've resolved it.